web/console/src/app/views/Division/index.tsx

```diff
@@ -11,21 +11,29 @@ import {
 } from '@/app/store/actions/divisions';
 import { DivisionClub } from '@/app/types/division';
 import { ToastNotifications } from '@/notifications/service';
-import { CurrentDivisionSeasons } from '@/divisions';
+
+import { DivisionsClient } from '@/api/divisions';
+import { DivisionsService } from '@/divisions/service';
 
 import realMadrid from '@static/img/divisions/realmadrid.png';
-import rectangle from '@static/img/FilterField/rectangle.svg';
 
 import './index.scss';
 
 const Division: React.FC = () => {
+    const DEFAULT_SEASONS_REWARD_TOKENS_STATUS = 0;
+
+    const divisionClient = new DivisionsClient();
+    const divisionService = new DivisionsService(divisionClient);
+    
```

Comment:
should not be here

Reply:
I add it because it does not matter to division entity, so i do not where should i add getting reward info